internal/output/html/style.css

```diff
@@ -495,26 +503,30 @@ header {
 
 .tooltip .tooltiptext {
   visibility: hidden;
-  width: 120px;
+
   background-color: black;
   color: #fff;
-  text-align: center;
-  border-radius: 4px;
-  padding: 5px 5px;
-  white-space: nowrap;
-  width: fit-content;
+
+  white-space: normal;
   text-align: left;
+  line-height: 1.5;
+
+  max-width: 300px;
```

Comment:
Setting max width is not good for all use cases:
![Screenshot 2025-02-03 at 5 17 26 PM](https://github.com/user-attachments/assets/35a4db62-ae03-4397-89a6-90de3f89fa7a)

Reply:
Also, are there any reasons for placing the tooltip above the text? I feel it's more common to position it below.